LethalAPI.Core/ModdedLobbyManager.cs

```diff
@@ -0,0 +1,47 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="ModdedLobbyManager.cs" company="LethalAPI Modding Community">
+// Copyright (c) LethalAPI Modding Community. All rights reserved.
+// Licensed under the GPL-3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace LethalAPI.Core;
+
+// ReSharper disable MemberCanBePrivate.Global
+// ReSharper disable once InconsistentNaming
+#pragma warning disable SA1401
+
+/// <summary>
+/// Manages the modded lobby instances.
+/// </summary>
+public static class ModdedLobbyManager
+{
+    /// <summary>
+    /// The count of mods installed.
```

Comment:
Should be changed to "the count of time an mod forced user to only join for modded lobby" or anything better than this. Because this counter not reflect the actual count of mods installed.

Reply:
Gotchya. I didn't know the exact implementation so I added this just to have a comment. The project will throw an error when commenting is missing, as to ensure our documentation is present.